lib/awful/tag.lua

```diff
@@ -795,6 +797,32 @@ function tag.incgap(add, t)
     tag.object.set_gap(t, tag.object.get_gap(t) + add)
 end
 
+--- The minimum number of client at which to enable gaps.
+--
+-- @beautiful beautiful.useless_gap_count
+-- @param integer (default: 0)
+-- @see gap
+-- @see gap_count
+
+--- The minimum number of client at which to enable gaps.
+--
+-- **Signal:**
+--
+-- * *property::gap\_count*
```

Comment:
Isn't it `property::useless_gap_count` due to using a different name internally?